modules/openapi-generator/src/main/java/org/openapitools/codegen/serializer/OpenAPISerializer.java

```diff
@@ -4,7 +4,7 @@
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
 import io.swagger.v3.oas.models.OpenAPI;
-
+import java.util.Map;
```

Comment:
if you can file a separate PR for changes in this file "OpenAPISerializer", I can try to get it merged more quickly.

Reply:
Yes will do right now